key-manager/src/main/scala/org/bitcoins/keymanager/KeyManager.scala

```diff
@@ -0,0 +1,136 @@
+package org.bitcoins.keymanager
+
+import java.nio.file.Files
+
+import org.bitcoins.core.compat.{CompatEither, CompatLeft, CompatRight}
+import org.bitcoins.core.crypto._
+import org.bitcoins.core.hd.{HDAccount, HDPath}
+import org.bitcoins.core.util.BitcoinSLogger
+import org.bitcoins.keymanager.util.HDUtil
+import scodec.bits.BitVector
+
+import scala.util.{Failure, Success, Try}
+
+case class KeyManager(
+    private val mnemonic: MnemonicCode,
+    kmParams: KeyManagerParams) {
+
+  private val seed = BIP39Seed.fromMnemonic(mnemonic, BIP39Seed.EMPTY_PASSWORD) // todo think more about this
```

Comment:
ping / maybe reference the issue in the comment

Reply:
https://github.com/bitcoin-s/bitcoin-s/issues/972#issuecomment-568460716